app/interfaces/api/v2/case_workers/claim.rb

```diff
@@ -52,19 +52,19 @@ def value_band_id
             end
 
             def current_claims
-              current_user.claims.caseworker_dashboard_under_assessment.search(
+              current_user.claims.search(
                 search_terms, Claims::StateMachine::CASEWORKER_DASHBOARD_UNDER_ASSESSMENT_STATES, *search_options
               )
             end
 
             def archived_claims
-              ::Claim::BaseClaim.active.caseworker_dashboard_archived.search(
+              ::Claim::BaseClaim.active.search(
                 search_terms, Claims::StateMachine::CASEWORKER_DASHBOARD_ARCHIVED_STATES, *search_options
               )
             end
 
             def allocated_claims
-              ::Claim::BaseClaim.active.__send__(scheme).caseworker_dashboard_under_assessment.search(
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2316/files#r180155331)